client/packages/common/src/ui/layout/tables/index.ts

```diff
@@ -33,3 +33,12 @@ export const DEFAULT_PAGE_SIZE = 25;
 export const placeholderRowStyle: AppSxProp = {
   color: theme => theme.palette.secondary.light,
 };
+
+export const textStyles = {
```

Comment:
This should be implemented on all the `custom` cells eventually ([like location](https://github.com/msupply-foundation/open-msupply/issues/7699)) to get styling right